__docs__/wonder-blocks-dropdown/option-item.stories.tsx

```diff
@@ -55,7 +55,9 @@ export default {
     decorators: [
         (Story): React.ReactElement<React.ComponentProps<typeof View>> => (
             <View style={styles.example}>
-                <Story />
+                <div role="listbox" aria-label="Example">
```

Comment:
Something I've been doing to address a11y warnings is to disable specific rules if we know it is expected.

 For example, for the `NavigationTabItem` stories, because it renders a `<li>` without a `<ul>` (since that's provided by `NavigationTabs`), we expect a related warning about that, but there should be no warnings when `NavigationTabs` and `NavigationTabItem` are used together! Here's an [example of disabling the `listitem` rule](https://github.com/Khan/wonder-blocks/pull/2500/files#diff-4f9d8a9178620afc2da5e873abed4bf2d97c9f7851030b313d503fbb59706679R18-R27) when it's expected

Do we have a preference for disabling expected rules or wrapping it with additional things to make sure it's semantically correct? Curious to see if others have thoughts on this!

Reply:
That's a really good question. My personal opinion is that it would be better to render an accessible example even if the component is incomplete by itself. The code could document those details so it's clear!

Reply:
Sounds good! I'll update those stories, thanks for the feedback!